ibm_catalog.json

```diff
@@ -562,8 +562,189 @@
               "optional": true
             }
           ],
+          "configuration": [
+            {
+              "key": "ibmcloud_api_key"
+            },
+            {
+              "key": "prefix"
+            },
+            {
+              "key": "cluster_id"
+            },
+            {
+              "key": "cluster_resource_group_id"
+            },
+            {
+              "key": "cluster_config_endpoint_type",
+              "options":[
+                {
+                  "displayname" : "default",
+                  "value" : "default"
+                },
+                {
+                  "displayname" : "private",
+                  "value" : "private"
+                },
+                {
+                  "displayname" : "vpe",
+                  "value" : "vpe"
+                },
+                {
+                  "displayname": "link",
+                  "value": "link"
+                }
+              ]
+            },
+            {
+              "key": "is_vpc_cluster"
+            },
+            {
+              "key": "wait_till"
+            },
+            {
+              "key": "wait_till_timeout"
+            },
+            {
+              "key": "log_analysis_enabled"
+            },
+            {
+              "key": "log_analysis_agent_tags"
+            },
+            {
+              "key": "log_analysis_add_cluster_name"
+            },
+            {
+              "key": "log_analysis_ingestion_key"
+            },
+            {
+              "key": "log_analysis_secret_name"
+            },
+            {
+              "key": "log_analysis_instance_region"
+            },
+            {
+              "key": "log_analysis_endpoint_type",
+              "options":[
+                {
+                  "displayname" : "private",
+                  "value" : "private"
+                },
+                {
+                  "displayname" : "public",
+                  "value" : "public"
+                }
+              ]
+            },
+            {
+              "key": "log_analysis_agent_custom_line_inclusion"
+            },
+            {
+              "key": "log_analysis_agent_custom_line_exclusion"
+            },
+            {
+              "key": "log_analysis_agent_name"
+            },
+            {
+              "key": "log_analysis_agent_namespace"
+            },
+            {
+              "key": "log_analysis_agent_tolerations"
+            },
+            {
+              "key": "cloud_monitoring_enabled"
+            },
+            {
+              "key": "cloud_monitoring_access_key"
```

Comment:
same comment as https://github.com/terraform-ibm-modules/terraform-ibm-observability-da/pull/167/files#r1787492298 - make this required here